### ISSUE TITLE
Android browser target be ignored.

### ISSUE THREAD

**Issue by lili21** [babel/babel-preset-env#268]
### config

```
{
  "presets": [
    ["env", {
      "targets": {
        "browsers": [ "Android >= 4" ]
      },
      "modules": false,
      "useBuiltIns": true,
      "debug": true
    }],
   ...
```

### debug info

<img width="322" alt="2017-04-11 4 26 54" src="https://cloud.githubusercontent.com/assets/4456380/24899678/b76e7c50-1ed3-11e7-85c1-03e004afd6af.png">

**Comment by existentialism** [comment]
Fix is out in [v1.4.0](https://github.com/babel/babel-preset-env/releases/tag/v1.4.0)!

**Comment by lili21** [comment]
Got it. thanks.
